src/display-likes.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-return-await */
+
```

Comment:
- [ ] Please don't disable linters instead you resolve them by following the link that comes with errors because we use eslint to write good codes so disabling it doesn't make much sense

Reply:
This issue is resolved. Nice job! :clap: